src/styles/blocks/calendar.scss

```
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: (7 * $box-size) + (6 * $box-gap) + (2 * 10);
+  padding: 10px;
+  gap: $box-gap;
+
+  @each $d, $i in $day {
```

`d`, `i`?
use more specific variables
For example:
```suggestion
  padding: 10px;
  gap: $box-gap;

  @each $day, $index in $weekDays {
```